etna/insights/models.py

```diff
@@ -0,0 +1,45 @@
+from django.db import models
+
+from wagtail.core.models import Page
+from wagtail.core.fields import StreamField
+from wagtail.admin.edit_handlers import FieldPanel, StreamFieldPanel
+
+
+from .blocks import InsightsPageStreamBlock
+
+
+class InsightsIndexPage(Page):
+    """InsightsIndexPage
+
+    This page lists the InsightsPage objects that are children of this page.
+    """
+    introduction = models.CharField(max_length=200, blank=False)
+
+    def get_context(self, request):
+        context = super().get_context(request)
+        insights_pages = self.get_children().live()
```

Comment:
This looks good. 

It's probably worth excluding private pages:

https://docs.wagtail.io/en/stable/reference/pages/queryset_reference.html#wagtail.core.query.PageQuerySet.public

Along with the specific page type instances:

https://docs.wagtail.io/en/stable/reference/pages/queryset_reference.html#wagtail.core.query.PageQuerySet.specific